web/themes/custom/pineapple/pineapple.theme

```diff
@@ -27,9 +27,10 @@ function pineapple_theme_suggestions_node_alter(array &$suggestions, array $vari
  * Implements hook_theme_suggestions_field_alter().
  */
 function pineapple_theme_suggestions_field_alter(array &$suggestions, array $variables) {
-  /*$element = $variables['element'];	  $suggestions[] = 'field--inline-label';
-$suggestions[] = 'field__' . $element['#view_mode'];
-$suggestions[] = 'field__' . $element['#view_mode'] . '__' . $element['#field_name'];*/
+  $element = $variables['element'];
+  if ($element['#label_display'] == 'inline') {
+    $suggestions[] = 'field--inline-label';
```

Comment:
You should be using underscores here as per the examples you removed and also instead of the word inline consider concatenating the label display variable used in your if statement also as per the examples in the code you removed

Reply:
okay, done.